File: app/src/main/java/trunk/doi/base/ui/fragment/info/mvp/InfoModel.java
package trunk.doi.base.ui.fragment.info.mvp;


import com.base.lib.integration.repository.IRepositoryManager;
import com.base.lib.mvp.BaseModel;

import com.base.lib.di.scope.FragmentScope;

import javax.inject.Inject;

import trunk.doi.base.ui.fragment.info.mvp.InfoContract;

/**
 * ================================================
 * Description:
 * <p>
 * Created by GasMvpFragment on 11/30/2019 14:54
 * ================================================
 */

@FragmentScope
public class InfoModel extends BaseModel implements InfoContract.Model {

    @Inject
    public InfoModel(IRepositoryManager repositoryManager) {
        super(repositoryManager);
    }

    @Override
    public void onDestroy() {
        super.onDestroy();

    }
}

File: app/src/main/java/com/gas/app/ui/fragment/info/mvp/InfoModel.kt
package com.gas.app.ui.fragment.info.mvp

import com.base.lib.di.scope.FragmentScope
import com.base.lib.integration.repository.IRepositoryManager
import com.base.lib.mvp.BaseModel
import javax.inject.Inject

/**
 * ================================================
 * Description:
 *
 *
 * Created by GasMvpFragment on 11/30/2019 14:54
 * ================================================
 */
@FragmentScope
class InfoModel @Inject constructor(repositoryManager: IRepositoryManager) : BaseModel(repositoryManager), InfoContract.Model {
    override fun onDestroy() {
        super.onDestroy()
    }
}